src/lib/elastic.js

```diff
@@ -124,7 +162,7 @@ function getClient (config) {
   }
 
   if (!esClient) {
-    esClient = new es.Client({ node, auth, apiVersion, requestTimeout })
+    esClient = new es.Client({ nodes, auth, apiVersion, requestTimeout, pingTimeout })
   }
```

Comment:
I would move this to the top of the function and would ask if the esClient is set already so we can skip all the code below. 

So at the first line of the function we would then call this code:

```ts
if (esClient) return esClient
```

Reply:
I've added it.